src/components/Label/Label.test.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Label, {LabelProps} from '.';
+
+describe('Label', () => {
+  const rgbWhite = 'rgb(255, 255, 255)';
+  const rgbBlack = 'rgb(0, 0, 0)';
+  const rgbDark = 'rgb(34, 34, 34)';
+  const rgbLight = 'rgb(221, 221, 221)';
+  const hexDark = '222';
+  const hexLight = 'ddd';
+
+  const baseProps: LabelProps = {
+    color: rgbWhite,
+    name: 'Hello world',
+  };
+
+  it('renders with name passed in', () => {
+    render(<Label {...baseProps} />);
+
+    expect(screen.getByTestId('Label')).toBeTruthy();
+    expect(
+      screen.getAllByText((_, node: Element) => {
```

Comment:
note that there is an issue where `getAllByText` ignores the inner span text: https://github.com/testing-library/dom-testing-library/issues/410